### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by jemisonf** [osulp/tresbien#77]
General issues:
- [ ] Test pages at 200, 300, and 400% zoom. Avoid scroll bars if possible, but pages should at least be usable
- [x] Pages should use consistent heading structure--use h2s for section headings, h3s for sub sections, etc. Don't skip a level (e.g. h2 -> h4)
- [x] Use html5 areas (`<nav>`, `<main>`, etc.)
- [x] All images should have succinct alt text
- [x] Add a 'skip to main content' link at the top of each page. Hide it when not selected by the keyboard.
- [x] Make sure text is large enough to be easily readable
- [ ] Make sure you can navigate each page using only the keyboard
- [ ] Links should be distinguished by something other than color. 
- [ ] Elements that are focused by the keyboard should have an obvious halo
- [x] Navbar takes up too much space on high zoom levels

Form issues:
- [ ] Use `aria-labelledby` to properly label table cells in main form. We should decide on a specific scheme we want to use for this.
- [ ] All buttons should function as buttons (press space to click) and links should function as links (press enter to click) and our link/button usage should be semantically accurate--a link directs to another page while a button does something on the current page. This could require some JS to work correctly with Cocoon add links.
- [x] Make sure tool tips are usable at high magnification
- [ ] Clearly identify fields that are required
- [x] Clearly identify fields that have an error
- [x] Rapidly tabbing through date fields leaves the datetimepicker dialogue open
